tests/cases/php-langspec/exception_handling/odds_and_ends.php

```diff
@@ -20,14 +20,22 @@ class X {}	// not derived from Exception
 }
 //*/
 ///*
-catch (int $e) {		// Not diagnosed; bad
-	echo "L0: In catch-block int\n";
-}
+//catch (int $e) {		// Diagnosed; good
+//	echo "L0: In catch-block int\n";
+//}
 //*/
 ///*
 catch (X $e) {		// Not diagnosed; bad
 	echo "L0: In catch-block int\n";
 }
+
+//catch (InvalidArgumentException | $e) { // Diagnosed; good
+//    echo "L0: In catch-block multiple Exception\n";
+//}
+
+catch (BadMethodCallException | InvalidArgumentException $e) {
+    echo "L0: In catch-block multiple Exception\n";
+}
```

Comment:
Yep, let's revert this file but just make sure that there are handwritten test cases added for this scenario (which it looks like you already added). The spec tests are generated using [this script](https://github.com/Microsoft/tolerant-php-parser/blob/master/tests/generate_spec_tests.php) and will be overwritten whenever we take updates from [php-langspec](github.com/php/php-langspec). Original source [here](https://github.com/Microsoft/tolerant-php-parser/blob/master/php-langspec/tests/exception_handling/odds_and_ends.phpt).